library/bdImage/bdApi/ControllerApi/Post.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class bdImage_bdApi_ControllerApi_Post extends XFCP_bdImage_bdApi_ControllerApi_Post
+{
+    public function actionGetEditorConfig()
+    {
+        $response = parent::actionGetEditorConfig();
+
+        if ($this->_isFieldIncluded('post_body_image') && ($response instanceof XenForo_ControllerResponse_View)) {
+            $input = $this->_input->filter(array('post_id' => XenForo_Input::UINT));
+            list($post, $thread, $forum) = $this->_getForumThreadPostHelper()->assertPostValidAndViewable($input['post_id']);
```

Comment:
`$thread` với `$forum` không xài thì để trống nha.

```suggestion
            list($post,,) = $this->_getForumThreadPostHelper()->assertPostValidAndViewable($input['post_id']);
```